src/libsyntax/ext/tt/macro_parser.rs

```diff
@@ -39,40 +39,40 @@
 //! Example: Start parsing `a a a a b` against [· a $( a )* a b].
```

Comment:
Missed `` `…` `` here. 

The current rendering of this doc comment is [totally broken](http://manishearth.github.io/rust-internals-docs/syntax/ext/tt/macro_parser/index.html). Perhaps better just ````` ```text … ``` ````` the whole thing.